config/webpack/environment.js

```diff
@@ -22,6 +22,10 @@ environment.splitChunks((config) => {
         ...(config.resolve ? config.resolve.alias : {}),
         '@crayons': path.resolve(__dirname, '../../app/javascript/crayons'),
         '@utilities': path.resolve(__dirname, '../../app/javascript/utilities'),
+        '@controllers': path.resolve(
+          __dirname,
+          '../../app/javascript/controllers',
+        ),
         '@components': path.resolve(
```

Comment:
The [webpack alias](https://webpack.js.org/configuration/resolve/#resolvealias) for controllers.

Reply:
@nickytonline is this config so that we can now use Stimulus controllers on the user facer side of the app (like the Creator Settings Controller)?

Reply:
No, this config just allows us to use an alias is all. It's more to avoid having to do e.g. `../../controllers/some-controller.js` vs. `@controllers/some-controller.js`.